generators/assets/webpack/templates/assets/robots.txt.tmpl

```diff
@@ -0,0 +1,2 @@
+User-agent: *
```

Comment:
We should probably generate this file for non-webpack apps as well.

Reply:
Right, just added that default robots to the non-webpack generated app.